src/implementations/vanilla/components/tabs/tabs.js

```diff
@@ -334,6 +374,100 @@ export class Tabs {
     this.moreButton.setAttribute('aria-expanded', false);
     this.dropdown.classList.remove('ecl-tabs__dropdown--show');
   }
+
+  /**
+   * Tabs onKeydown handler.
+   * @param {Event} e
+   */
+  onKeydown(e) {
+    const tgt = e.currentTarget;
+    let flag = false;
+
+    switch (e.key) {
+      case 'ArrowLeft':
+        this.moveFocusToPreviousTab(tgt);
+        flag = true;
+        break;
+
+      case 'ArrowRight':
+        this.moveFocusToNextTab(tgt);
+        flag = true;
+        break;
+
+      case 'Home':
+        this.moveFocusToTab(this.firstTab);
+        flag = true;
+        break;
+
+      case 'End':
+        this.moveFocusToTab(this.lastTab);
+        flag = true;
+        break;
+
+      default:
+        break;
```

Comment:
never seen a break on a default case for a switch, what would that be doing..? :)